compiler/rustc_typeck/src/impl_wf_check/min_specialization.rs

```diff
@@ -179,7 +181,9 @@ fn get_impl_substs<'tcx>(
 ///
 /// For example given the impl:
 ///
+/// ```rust
 /// impl<'a, T, I> ... where &'a I: IntoIterator<Item=&'a T>
+/// ```
```

Comment:
This is invalid syntax.
```suggestion
/// ```text
/// impl<'a, T, I> ... where &'a I: IntoIterator<Item=&'a T>
/// ```
```